cmds/boot/boot/boot.go

```diff
@@ -89,7 +89,7 @@ func main() {
 	if *verbose {
 		l = ulog.Log
 	}
-	images, mps, err := localboot.Localboot(l, blockDevs)
+	images, mountPool, err := localboot.Localboot(l, blockDevs)
```

Comment:
in my heart of hearts I wish this were not part of this PR. It's unrelated to the subject.

Reply:
If we get some general agreement that the mount.Pool is a good approach for this, I can split it off into a separate PR to simplify the PRs (github isn't as nice as gerrit for patch series I'm afraid).

Reply:
I broke this off into #1860